src/rendering/RenderUtil.cc

```diff
@@ -2230,6 +2364,104 @@ void RenderUtilPrivate::LowlightNode(const rendering::NodePtr &_node)
   }
 }
 
+/////////////////////////////////////////////////
+void RenderUtil::ViewInertia(const Entity &_entity)
+{
+  std::vector<Entity> inrLinks;
```

Comment:
Try to use the hole name
```suggestion
  std::vector<Entity> inertiaLinks;
```

Reply:
Fixed in [`9be5a99`](https://github.com/ignitionrobotics/ign-gazebo/pull/861/commits/9be5a9997478fe66a18f826ec83017d54c39fb22)